manage_runner.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+url='https://gitlab.com/'
+token='a-oX5iEQx5D5rdyA8Bxu'
+name='gcc_runner'
+executor='virtualbox'
+VM='Centos-68-64bit'
+user='rpm_omnibus'
+password='1ns3cur3'
+
+usage() {
+  cat <<EOF
+  usage: $0 -m value
+  This script registers/unregisters $VM VM as a gitlab runner for a repository
+  with token=$token.
+  OPTIONS:
+  -h         Show this message
+  -m value   Register/unregister the VM as a gitlab runner
```

Comment:
You don't need "value" here, giving that you have already written it in the "usage: ..." line